python/src/opendp/_extrinsics/synth/__init__.py

```diff
@@ -0,0 +1,61 @@
+from opendp.mod import Domain, Metric
+import opendp.prelude as dp
+
+
+class Synthesizer:
+    def __init__(self):
+        self._is_fitted = False
+
+    def fit(self, data):
+        assert not self._is_fitted, "Synthesizer is already fitted"
+
+    def sample(self, num_records):
+        assert self._is_fitted, "Synthesizer must be fitted first"
+
+    def releasable(self):
+        assert self._is_fitted, "Synthesizer must be fitted first"
+
+
+class IdentitySynthesizer(Synthesizer):
+    def __init__(self, epsilon):
+        super().__init__()
+
+    def fit(self, data):
+        super().fit(data)
+        self._is_fitted = True
+        self._data = data
+
+    def sample(self, num_records):
+        super().sample(num_records)
+        return self._data.sample(num_records)
+
+
+def make_private_synthesizer_trainer(input_domain: Domain,
+                                     input_metric: Metric,
+                                     epsilon: float,
+                                     name: str,
+                                     **kwargs):
+
+    print("THIS IS A PLACEHOLDER. NOT DIFFERENTIALLY PRIVATE!!!")
```

Comment:
Placeholder in the sense that this is a TODO? Or is this code that is working correctly, but we want to be sure users don't do the wrong thing with it?